dev/integration_tests/flavors/macos/Runner.xcodeproj/xcshareddata/xcschemes/paid.xcscheme

```diff
@@ -14,24 +14,23 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "F383D4A52938029D00598432"
-               BuildableName = "Paid App.app"
-               BlueprintName = "Paid App"
+               BlueprintIdentifier = "33CC10EC2044A3C60003C045"
+               BuildableName = "Debug Free.app"
+               BlueprintName = "Runner"
                ReferencedContainer = "container:Runner.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug Paid"
+      buildConfiguration = "Debug-free"
```

Comment:
I messed up the ordering of things here, hence weird diffs like these where "paid" got swapped out with "free".

As for swapping ` <flavor>` with `-flavor`, see dev/integration_tests/flavors/macos/Runner.xcodeproj/project.pbxproj:

> The build configurations in the free scheme have the `-free` suffix.